values/values.go

```diff
@@ -148,6 +151,16 @@ func (v value) Equal(r Value) bool {
 	}
 }
 
+func (v value) Retain() {
```

Comment:
I'm a bit confused by this. To my knowledge, value is only used by primitive types so it shouldn't need retain and release. I consider it a bit suspect that this calls New and then retain or release. That doesn't make much sense to me.

Reply:
You're right. I think this is the result of me just misunderstanding the code. I'll change these to be no-ops.